bin/ezmlm-send.1

```diff
@@ -41,7 +41,7 @@ to create a subject index from a preexisting archive.
 Subject and author lines are decoded if they are encoded per rfc2047. When
 split lines are unfolded, the number of escape sequences for
 iso-2022-* character sets is minimized. For instance, two
-consequtive toascii sequences are reduced.
+consecqutive toascii sequences are reduced.
```

Comment:
oops, the correct spelling here is `consecutive`

Reply:
Sigh, serves me right for not letting my browser suggest the right spelling.

Reply:
Isn't there an axiom that everyone who corrects another's spelling/grammar on the internet makes an error of their own?

Reply:
I've definitely used https://github.com/jsoref/spelling to find bugs in other spell checkers...

Someday I'll add a script which spellchecks my commit messages since it should highlight places where I've made typos...